test/testSanity.js

```diff
@@ -0,0 +1,15 @@
+var expect = require('chai').expect;
```

Comment:
> when we enable the linter for tests it will yield for this and a few other things

What does it mean for the linter to yield? Will it continue to process the rest of the file?